### PR TITLE
Add support for customising a card title color

### PR DESCRIPTION
**Pull request by MitchellCash**
Currently the card title will use the card body colour for its text. This pull requests lets you set a different card title colour compared to the body, which I find useful especially when using the new CSS variable `--#{$prefix}card-title-color`.

Personally I prefer this approach versus adding a utility class like `text-primary` as it allows for greater customisation via the SASS/CSS variables.

 If this is useful I can also add a commit for the card subtitle?

### CODE REVIEW

**Comment by tungphan995** [comment]
Thank you for believing in me.

File: scss/_card.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@
   --#{$prefix}card-cap-padding-x: #{$card-cap-padding-x};
   --#{$prefix}card-cap-bg: #{$card-cap-bg};
   --#{$prefix}card-cap-color: #{$card-cap-color};
+  --#{$prefix}card-title-color: #{$card-title-color};
```
Comment (julien-deramond) [inline scss/_card.scss:19]: Could you please regroup it with the other `--#{$prefix}card-title-*` in the file?

File: scss/_variables.scss [hunk 2]
```diff
@@ -1245,6 +1245,7 @@ $card-cap-padding-y:                $card-spacer-y * .5 !default;
 $card-cap-padding-x:                $card-spacer-x !default;
 $card-cap-bg:                       rgba($black, .03) !default;
 $card-cap-color:                    null !default;
+$card-title-color:                  null !default;
```
Comment (julien-deramond) [inline scss/_variables.scss:1248]: Could you please regroup it with the other `$card-title-*` in the file?

**Review by julien-deramond — CHANGES_REQUESTED** [review]
Sorry for the delay @MitchellCash. We were working on the v5.2* until now :)

On my side, I find this request legitimate since we already have `--bs-card-color` and `--bs-card-cap-color` so let's go for more customization. In the same spirit, adding the same thing for card subtitle color might be useful as well.

@mdo Thoughts about this change? Any strong opinions against it?

File: scss/_card.scss (hunk 1 above)
Comment (MitchellCash) [reply scss/_card.scss]: Fixed in https://github.com/twbs/bootstrap/pull/36979/commits/41419dfab6f4b22e0b1970700727ffa21525e611.

File: scss/_variables.scss (hunk 2 above)
Comment (MitchellCash) [reply scss/_variables.scss]: Fixed in https://github.com/twbs/bootstrap/pull/36979/commits/41419dfab6f4b22e0b1970700727ffa21525e611.

**Review by julien-deramond — APPROVED** [review]

**Comment by MitchellCash** [comment]
Ready for review/merge now 😃

**Review by mdo — APPROVED** [review]

**Review by mdo — APPROVED** [review]

**Pull request merged**